n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -69,14 +69,14 @@ def test_eko_utils(tmp_path):
     assert (
         t_card_dict["order"][0] == pto + 1
     )  # This is due to a different convention in eko orders due to QED
-    assert_allclose(op_card_dict["rotations"]["xgrid"], x_grid)
-    assert_allclose(op_card_dict["_mugrid"][0], 1.65)
-    assert_allclose(op_card_dict["_mugrid"][-1], q_fin)
+    assert_allclose(op_card_dict["xgrid"], x_grid)
+    assert_allclose(op_card_dict["mugrid"][0], (1.65, 4))
+    assert_allclose(op_card_dict["mugrid"][-1], (q_fin, 5))
```

Comment:
What does the and 4 and 5 mean here? (for my own understanding of the changes)

Reply:
the number of active flavors. Now for each Q2 you need to provide also the desired nf